sdk/core/azure-core/src/main/java/com/azure/core/util/tracing/Tracer.java

```diff
@@ -40,61 +38,102 @@
     String HOST_NAME = "hostname";
 
     /**
-     * Key for {@link Context} which indicates that the context contains message Span Context.
+     * Key for {@link Context} which indicates that the context contains a message span context.
      *
      */
     String SPAN_CONTEXT = "span-context";
 
     /**
-     * Key for {@link Context} which indicates that the context contains the Diagnostic Id for the service call.
+     * Key for {@link Context} which indicates that the context contains a "Diagnostic Id" for the service call.
      *
      */
     String DIAGNOSTIC_ID_KEY = "diagnostic-id";
 
     /**
      * Creates a new tracing span.
      *
-     * The {@code context} will be checked for containing information about a parent span. If a parent span is found the
-     * new span will be added as a child, otherwise the span will be created and added to the context and any downstream
-     * start calls will use the created span as the parent.
+     * The {@code context} will be checked for containing information about a parent span. If a parent span is found,
+     * the new span will be added as a child. Otherwise the parent span will be created and added to the {@code context}
+     * and any downstream {@code start()} calls will use the created span as the parent.
+     *
+     * <p><strong>Code samples</strong></p>
+     *
+     * <p>Starts a tracing span with provided method name and explicit parent span</p>
+     * {@codesnippet com.azure.core.util.tracing.start#string-context}
      *
      * @param methodName Name of the method triggering the span creation.
      * @param context Additional metadata that is passed through the call stack.
-     * @return An updated context object.
+     *
+     * @return The updated {@link Context} object containing the returned span.
+     * @throws NullPointerException if {@code methodName} or {@code context} is {@code null}.
      */
     Context start(String methodName, Context context);
 
     /**
-     * Creates a new tracing span.
+     * Creates a new tracing span for AMQP calls.
      *
      * The {@code context} will be checked for containing information about a parent span. If a parent span is found the
```

Comment:
Also wrap in a paragraph tag, remember Javadocs are HTML so newlines are ignored unless it's a pre tag.